### ISSUE TITLE
Quicklisp installation?

### ISSUE THREAD

**Issue by swapneils** [VitoVan/calm#177]
I see CALM recently got added to QuickLisp (in the october release). 

While the `calm` CLI obviously wouldn't be added to the path (unless added programmatically via eg `uiop` calling shell commands), is the Quicklisp distribution otherwise usable? And if so can we add a note in the README so that people know it's an option?

**Comment by VitoVan** [comment]
You are right, we should add a note in the README, to clarify how to use the Quicklisp version.
